apiserver/undertaker/undertaker.go

```diff
@@ -85,28 +85,16 @@ func (u *UndertakerAPI) RemoveModel() error {
 	return u.st.RemoveAllModelDocs()
 }
 
-func (u *UndertakerAPI) environResourceWatcher() params.NotifyWatchResult {
+func (u *UndertakerAPI) modelEntitiesWatcher() params.NotifyWatchResult {
 	var nothing params.NotifyWatchResult
-	machines, err := u.st.AllMachines()
-	if err != nil {
-		nothing.Error = common.ServerError(err)
-		return nothing
-	}
-	services, err := u.st.AllApplications()
+
+	m, err := u.st.Model()
```

Comment:
oh k.. I was looking at other code in the area. I'll take a look to see what state.ModelUUID actually gives here :)